src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -63,7 +63,7 @@ class ConfirmationPage extends React.Component {
             <strong>Conditions claimed</strong>
             <br/>
             <ul className="disability-list">
-              {selectedDisabilities.map((disability, i) => {
+              {disabilities.map((disability, i) => {
```

Comment:
Will these be filtered to only display the selected disabilities?

Reply:
Yeah, that's a good idea. It's not really part of this PR, but it's an easy enough change.

```js
disabilities.filter(d => d['view:selected']).map(...)
```

Reply:
Good thinking; probably needs its own issue.